source/mainnet/net/guides/run-node-aws.rst

```diff
@@ -0,0 +1,165 @@
+.. _Discord: https://discord.gg/xWmQ5tp
+
+.. _run-a-node-aws:
+
+=================
+Run a node on AWS
+=================
+
+In this guide, you will learn how to setup a node on an AWS VPS, using the marketplace product.
```

Comment:
**Suggestion (non-blocking):** Again it would be good to introduce the abbreviation `AWS VPS`.